nhost/compose/config.go

```diff
@@ -126,9 +78,15 @@ func NewConfig(conf *nhost.Configuration, p *ports.Ports, env []string, gitBranc
 	return &Config{nhostConfig: conf, ports: p, dotenv: env, gitBranch: gitBranch, composeProjectName: projectName}
 }
 
-func (c Config) addLocaldevExtraHost(svc *types.ServiceConfig) *types.ServiceConfig {
+func (c Config) addExtraHosts(svc *types.ServiceConfig) *types.ServiceConfig {
```

Comment:
I am not sure we need to set all hosts for each service if the value for all fields is always the same.